internal/server/v1/dlq/service.go

```diff
@@ -24,3 +28,16 @@ func NewService(client entropyv1beta1rpc.ResourceServiceClient, gcsClient gcs.Bl
 		cfg:       cfg,
 	}
 }
+
+func (s *Service) getDlqJob(ctx context.Context, jobURN string) (*models.DlqJob, error) {
```

Comment:
let's use capital `GetDlqJob` to make this function as public.
This will be needed for testing.

Reply:
fixed